django-pagination/terms/templates/terms/base.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Python Keywords</title>
+    <link rel="stylesheet" href="https://cdn.simplecss.org/simple.min.css">
```

Comment:
:wrench: **Technical Recommendation**

I'd recommend using a local CSS file instead of relying on an external source, which might go out of service. Besides, readers without Internet access won't be able to run your code and see the expected result.